### DISCUSSION TITLE
請問一下老師或各位大大知道這個網站時間條不能動是怎麼做到的嗎?

### DISCUSSION THREAD

**Discussion by 109302013evantsao** [1101-GS4719-C/Python-Community-GS4719-C#4] [Q&A]
如題 下列這個網站一進入後就無法撥動時間條
不知道這個網頁是使用什麼特殊的寫法
[網站](https://www.thisworldthesedays.com/6-ways-to-make-yourself-feel-better-right-now.html?fbclid=IwAR0mjfOPx-LFej2w0H3NJ5uzOLzKAoMdyyAhvsbrTuWzfnCccR-zcQMxfeM)

**Comment by enoch-lightning** [comment]
雖然目前我不知道這個網站「無法控制時間」是由什麼程式的什麼語法達成。就我目前的理解，寫「網頁=Web」的程式，可以用如下幾個程式語言達成：
1.HTML
2.CSS
3.JavaScript
4.Node.JS
5.Vue.JS
6.Python
7.Flask
上面的程式，好像彼此有些相關。此處的分類，完全將其分開。分類如有錯誤，想請老師提點。
我想，不管是哪一種語言，都可以做到「無法控制時間」這件事。因為，我覺得這就是「不想讓你調整時間」的設定，所以「權限」的設定可能是重點。

**Reply by chia-kai-chang** [reply]
> 雖然目前我不知道這個網站「無法控制時間」是由什麼程式的什麼語法達成。就我目前的理解，寫「網頁=Web」的程式，可以用如下幾個程式語言達成：
> 1.HTML
> 2.CSS
> 3.JavaScript
> 4.Node.JS
> 5.Vue.JS
> 6.Python
> 7.Flask
> 上面的程式，好像彼此有些相關。此處的分類，完全將其分開。分類如有錯誤，想請老師提點。
> 我想，不管是哪一種語言，都可以做到「無法控制時間」這件事。因為，我覺得這就是「不想讓你調整時間」的設定，所以「權限」的設定可能是重點。

來說一下分類：
1. HTML、CSS：寫靜態網頁的文件排版，就像是在編輯Word文件一樣
2. JavaScript：可以做到網頁的即時互動，不需要reload網頁也可以做到互動效果。此外，Node.js, Vue, React, Angular都是用Javascript來編寫的。目前我認為Javascript是跟Python一樣重要的語言
3. Node.js：做網頁後端用的，是用Javascript來編寫的
4. Flask框架：也是做網頁後端用的，它使用Python來編寫
5. Angular：網頁前端框架，是由Google開發的
6. React：網頁前端框架，是由Facebook開發的，目前學習CP值最高的前端框架
7. Vue：網頁前端框架，是由中國籍Google前員工離職後，建立的框架，目前為對岸最流行的前端框架

**Reply by enoch-lightning** [reply]
謝謝老師。雖然還不是很理解，但我先整理起來。

**Comment by chia-kai-chang** [comment]
1. 在影片按右鍵，選擇「檢查」；或是，在影片按右鍵，再按下鍵盤的「N」鍵
![image](https://user-images.githubusercontent.com/1768891/134108621-2308021a-496d-46f0-a7b6-6294e8365ab5.png)

2. 將看到以下的程式碼
![image](https://user-images.githubusercontent.com/1768891/134108795-6577e5ad-0e91-4c9d-8a30-96da7b368021.png)

3. 會發現 `<div id="vid-blocker"></div>` 層，把影片 **蓋住** 了。只要把這一個「層」刪除，或是修改上下排列的順序，就可以控制影片播放了。也就是說，之前不能播放的效果，是因為滑鼠作用在 `<div id="vid-blocker"></div>` 層上，而沒有作用在 youtube video上。

![image](https://user-images.githubusercontent.com/1768891/134109206-b655bab5-e58a-45a6-8a6d-d65dcbe0d435.png)

**Comment by AngusGuei** [comment]
還沒遇到這類問題 但先存起來再說XD

**Comment by pandaiscollin** [comment]
不僅能在被rick roll後找到問題上來問  還順便rick roll點進去網站的人  樓主你真行LUL

**Reply by chia-kai-chang** [reply]
原來我被RickRoll了~ QQ
https://youtu.be/zKh5ELjjuWk
